alacritty/src/logging.rs

```diff
@@ -129,6 +129,26 @@ impl log::Log for Logger {
     fn flush(&self) {}
 }
 
+fn create_log_message(record: &log::Record<'_>, target: &str) -> String {
+    let now = time::strftime("%F %T.%f", &time::now()).unwrap();
+    let mut msg = format!("[{}] [{:<5}] [{}] ", now, record.level(), target);
+
+    // Alignment for the lines after the first new line character in the payload. We don't deal
+    // with fullwidth/unicode chars here, so just `msg.len()` will work fine.
```

Comment:
```suggestion
    // with fullwidth/unicode chars here, so just `msg.len()` is sufficient.
```

I don't like the word "fine", it implies that it's not great. But we've already determined that this should actually be the best solution here, so I'd rather make that clear.